xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/breadcrumbs.less

```diff
@@ -19,8 +19,15 @@
 
     // Styles for the tree navigation.
     &.dropdown {
-      > .dropdown-toggle {
+      /* We make sure the dropdown toggle carets and the links get the same colors. */
+      & > .dropdown-toggle, & > a {
         color: @breadcrumb-color;
+        &:hover, &:focus {
```

Comment:
What about the `:active` and `:visited` states?

Reply:
There's no specific style for the `active` and `visited` states. Checked directly on my local distribution.
Note that if there was, we'd probably need to override any style added by the `visited` state, because it doesn't make sense for the buttons next to the links.

![image](https://github.com/user-attachments/assets/350f5af2-b41c-4df2-8262-9d482dbb9009)